### PR TITLE
Update document.csv

### PR DESCRIPTION
**Pull request by Janeishly**
Coraa ny Gael - fixed typos

### CODE REVIEW

File: OpenData/secular printed material - 1921 - 1990/Coraa ny Gael/Coraa ny Gael 1953-09-15/document.csv [hunk 1]
```diff
@@ -1,16 +1,16 @@
...
+"Va Mnr. Juan Thobm Kaighin, Balleygarret, Skyll Vreeshey, kaire feed blein as ’nane jeig d’eash er y cheid laa jeh’n vee shoh! ","     Mr. John Tom Kaighin, Ballagarret, Bride, was ninety-one years old on the first of this month! ",
 Ta mee sheiltyn d’el eh-hene gaase ny ’saa gagh vlein!,I guess that he himself is growing younger every year!,
-"Ta shin ooilley fo lhiastnys vooar dasyn son ooilley’n Ghailck vie, skeeaylyn as bealedys [beealydys], creenaght as tushtey, ta, dy-kinjagh deayrtey magh ass y veal echey goll-rish struan!","We are all in great debt to him for all the good Manx, stories and folklore, wisdom and knowledge, that always flows forth from his mouth like a stream!",
+"Ta shin ooilley fo lhiastnys vooar dasyn son ooilley’n Ghailck vie, skeeaylyn as bealedys [beealydys], creenaght as tushtey, ta, dy-kinjagh deayrtey magh ass y veal echey goll-rish struan!","We are all in great debt to him for all the good Manx stories and folklore, wisdom and knowledge, that always flows forth from his mouth like a stream!",
```
Comment (david-allison) [inline OpenData/secular printed material - 1921 - 1990/Coraa ny Gael/Coraa ny Gael 1953-09-15/document.csv:8]: This is a translation from:

> Ghailck vie, skeeaylyn as bealedys

"good Manx" is a separate element in the list

```suggestion
"Ta shin ooilley fo lhiastnys vooar dasyn son ooilley’n Ghailck vie, skeeaylyn as bealedys [beealydys], creenaght as tushtey, ta, dy-kinjagh deayrtey magh ass y veal echey goll-rish struan!","We are all in great debt to him for all the good Manx, stories and folklore, wisdom and knowledge, that always flows forth from his mouth like a stream!",
```

**Review by david-allison — APPROVED** [review]

**Comment by david-allison** [comment]
Thanks!!

File: OpenData/secular printed material - 1921 - 1990/Coraa ny Gael/Coraa ny Gael 1953-09-15/document.csv (hunk 1 above)
Comment (Janeishly) [reply OpenData/secular printed material - 1921 - 1990/Coraa ny Gael/Coraa ny Gael 1953-09-15/document.csv]: Why am I making so many mistakes with this proofreading? I do this all the time, and I'm fairly sure I don't misread things this often normally. Sorry!

Comment (lauterb) [reply OpenData/secular printed material - 1921 - 1990/Coraa ny Gael/Coraa ny Gael 1953-09-15/document.csv]: I wouldn't worry too much about it, Jane, probably the extra brain power of looking at the Manx, even out of the corner of your eye, makes it just a little bit more tricky.

Ken

Comment (Janeishly) [reply OpenData/secular printed material - 1921 - 1990/Coraa ny Gael/Coraa ny Gael 1953-09-15/document.csv]: True!

**Pull request merged**